ports/raspberrypi/boards/42keebs_frood/pins.c

```diff
@@ -43,6 +43,11 @@ STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_D15), MP_ROM_PTR(&pin_GPIO15) },
     { MP_ROM_QSTR(MP_QSTR_D16), MP_ROM_PTR(&pin_GPIO16) },
 
+    { MP_ROM_QSTR(MP_QSTR_D17), MP_ROM_PTR(&pin_GPIO17) },  // User LED
+    { MP_ROM_QSTR(MP_QSTR_LED), MP_ROM_PTR(&pin_GPIO17) },
+    { MP_ROM_QSTR(MP_QSTR_D19), MP_ROM_PTR(&pin_GPIO17) },  // VBUS sensing
```

Comment:
Can you confirm that GPIO17 should really be available as all these names?

Reply:
Hey, thanks for catching that. It was a stupid copy&paste mistake. Fixed.

By the way, is it OK to give each pin these two names (a generic one and a "friendly" one), or should I rather choose one?

Reply:
yes, it's fine. The one that appears first will be used when printing the pin, so just keep that in mind when writing the pin definition.  For instance, on pygamer, LED and D13 are the same, and both print as board.LED:
```
>>> board.D13
board.LED
```
this is due to the order in the pins file:
```
    { MP_OBJ_NEW_QSTR(MP_QSTR_LED),  MP_ROM_PTR(&pin_PA23) },
    { MP_OBJ_NEW_QSTR(MP_QSTR_D13),  MP_ROM_PTR(&pin_PA23) },
```

Reply:
.. so as you've written it, board.LED would print as D17, which is the reverse of what we usually do.

Reply:
Ah, got it, thanks for the explanation! Let me fix it quickly.